src/shogun/base/SGObject.cpp

```diff
@@ -698,6 +698,13 @@ CSGObject* CSGObject::clone()
 		const BaseTag& tag = it.first;
 		const Any& own = it.second.get_value();
 
+		if (!own.cloneable())
+		{
+			SG_SDEBUG(
+			    "Skipping clone of %s::%s of type %s.\n", this->get_name(),
+			    tag.name().c_str(), own.type().c_str());
```

Comment:
continue!

Reply:
Yeah but since it is here such thing might be useful anyway